### PR TITLE
PerformanceResourceTiming StartTime implementation

### PR DESCRIPTION
**Pull request by aug2uag**
StartTime necessary to calculate `duration`, cherry-picked https://github.com/davidweitzenfeld/servo/pull/1 

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because testing specifications need to be acquired

### CODE REVIEW

**Comment by aug2uag** [comment]
heads up the purpose of this was for issue #21255 where request for test specification was made, this PR was intended for the purposes of that issue-- thanks

File: doc/a3.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
+https://github.com/servo/servo/issues/21254
```
Comment (hbina) [inline doc/a3.md]: This file should not be included I think?

**Comment by jdm** [comment]
The start time attribute was added in #24508. Sorry, I missed that this work was going on in two places. We can calculate duration now, though!

**Pull request closed without merging**